api/clients/eigenda_client_e2e_test.go

```diff
@@ -29,7 +28,9 @@ func TestClientUsingTestnet(t *testing.T) {
 
 	t.Run("PutBlobWaitForConfirmationDepth0AndGetBlob", func(t *testing.T) {
 		t.Parallel()
-		logger := log.NewLogger(log.NewTerminalHandler(os.Stdout, true))
+		logger, err := logging.NewZapLogger(logging.Development)
```

Comment:
use slogger, not zaplogger (zaplogger should be deprecated! is it not @shrimalmadhur ?)

Reply:
there is slogger support in sdk: https://github.com/Layr-Labs/eigensdk-go/blob/dev/logging/slog_logger.go 

plugged zap through tests since it was easy to construct. On second glance though there is a `test_utils` function that generates a compatible logger specifically for testing. Using this instead

Reply:
[80bae61](https://github.com/Layr-Labs/eigenda/pull/1183/commits/80bae61cf5381bfc00edf173c8272ed3ad24b8b9)

Reply:
Nice thanks. I know there's slogger support in sdk, I wrote it :P
is the logging.NewSLogger not easy to use?
Didn't even notice this was just for tests. Agree its less important, but still better to be consistent and use slogger everywhere. Thanks for teh change